lib/metanorma/compile.rb

```diff
@@ -252,8 +256,54 @@ def process_extensions(extensions, file, isodoc, options)
       end
     end
 
+    def install_fonts(options)
+      if options[:no_install_fonts]
+        Util.log("[fontist] Skip font installation because" \
+          " --no_install_fonts argument passed", :debug)
+        return
+      end
+
+      manifest = @processor.fonts_manifest
+      if manifest.nil?
+        Util.log("[fontist] Skip font installation because font_manifest is missing", :debug)
+        return
+      end
+
+      agree_to_terms = options[:agree_to_terms] || false
+      continue_without_fonts = options[:continue_without_fonts] || false
+
+      install_fonts_safe(manifest, agree_to_terms, continue_without_fonts)
+    end
+
     private
 
+    def install_fonts_safe(manifest, agree, continue)
+      fontist_install(manifest, agree)
+    rescue Fontist::Errors::LicensingError
+      if continue
+        Util.log("[fontist] Processing will continue without fonts installed", :debug)
+      else
+        Util.log("[fontist] Aborting without proper fonts installed," \
+          " make sure that you are passed --agree_to_terms", :fatal)
+      end
+    rescue Fontist::Errors::NonSupportedFontError => e
+      font = /Font '([^']+)'/.match(e.to_s)[1]
+      Util.log("[fontist] '#{font}' font is not supported. " \
+        "Please go to github.com/metanorma/metanorma-#{@processor.short}/issues" \
```

Comment:
"Please report this issue at github.com/metanorma/metanorma-#{@processor.short}/issues"